data/sql/updates/pending_db_world/rev_1685926790862483900.sql

```diff
@@ -0,0 +1,25 @@
+-- Fix GambeObjects 180772 - 180869 - 180874 (Cluster Launcher) --
+UPDATE `gameobject_template` SET `AIName` = 'SmartGameObjectAI' WHERE `entry` = 180772;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 1 AND `entryorguid` = 180772);
+INSERT INTO `smart_scripts` VALUES
+(180772, 1, 0, 0, 8, 0, 100, 0, 26521, 0, 0, 0, 0, 11, 26522, 2, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Cluster Launcher - On Spellhit \'Lucky Lunar Rocket\' - Cast \'Lunar Fortune\'');
+
+
+UPDATE `gameobject_template` SET `AIName` = 'SmartGameObjectAI' WHERE `entry` = 180869;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 1 AND `entryorguid` = 180869);
+INSERT INTO `smart_scripts` VALUES
+(180869, 1, 0, 0, 8, 0, 100, 0, 26521, 0, 0, 0, 0, 11, 26522, 2, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Cluster Launcher - On Spellhit \'Lucky Lunar Rocket\' - Cast \'Lunar Fortune\'');
+
+UPDATE `gameobject_template` SET `AIName` = 'SmartGameObjectAI' WHERE `entry` = 180874;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 1 AND `entryorguid` = 180874);
+INSERT INTO `smart_scripts` VALUES
```

Comment:
For this ones and the ones above. Don't shorten INSERTS, always add all fields after VALUES

Reply:
Sorry I forgot to respond. The "Makes it more readable" change refers to less INSERTSs, this is to make sure all fields of smart_scripts are present:

```
(`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
```

Reply:
@Gultask what's the status of this PR? 

> The "Makes it more readable" change refers to less INSERTS

this would be a nice-to-have but I wouldn't hold this PR only for it, entity-grouped SQL blocks are acceptable too (it's just like Keira3 generates them by default)

Reply:
@Gultask my bad I didn't get the whole point, I updated this PR myself (but I didn't test it). Please have another look at it.